stats.go

```diff
@@ -15,24 +16,25 @@ type streamStats struct {
 	mutex       sync.Mutex
 	streamStart time.Time
 	streamLive  bool // True if live
-	viewers     map[string]string
+	viewers     map[*sessions.Session]int
```

Comment:
I don't like the idea of using a pointer as a map key.  A better idea would be to just use the ID of the session instead (from `Session.ID`).

Reply:
Do you prefer a copy of the `ID` or a pointer on the `ID` ?